src/services/ui/src/pages/welcome/index.tsx

```diff
@@ -176,10 +178,17 @@ export const Welcome = () => {
         <div className="flex justify-around items-center text-xl py-10 px-10 py-4 max-w-screen-xl mx-auto">
           <h4>Do you have questions or need support?</h4>
           <Button asChild>
-            <Link to="/faq">View FAQ</Link>
+            <Link
+              to="/faq"
```

Comment:
Anytime we're using the path `/faq` we should utilize the constant value `ROUTES.FAQ` to ensure consistency across the app